index.js

```diff
@@ -57,22 +57,33 @@ module.exports = {
       return this._super.treeForApp.call(this, tree);
     }
   },
-
+  // We manually invoke Babel for treeForAddon, treeForTestSupport and treeForAddonTestSupport
```

Comment:
Is this true for `treeForTestSupport`? It looks like we're both invoking Babel _and_ calling `_super` there.

The default `treeForTestSupport` implementation in ember-cli looks like a no-op, so its Babel transpilation must be managed elsewhere (which is fine for us, since the `test-support` tree doesn't acquire the addon namespace), meaning we can probably skip Babel for that one.

Reply:
removed babel